pkg/ingress/controller/controller.go

```diff
@@ -695,7 +703,12 @@ func (c *Controller) ensureIngress(ing *nwv1.Ingress) error {
 	}
 
 	// Create listener
-	listener, err := c.osClient.EnsureListener(resName, lb.ID, secretRefs)
+	sourceRanges := getStringFromIngressAnnotation(ing, IngressAnnotationSourceRangesKey, "0.0.0.0/0")
+	listenerAllowedCIDRs := strings.Split(sourceRanges, ",")
+	if err != nil {
```

Comment:
I didn't see there is any err raised above.

Reply:
Fixed by https://github.com/kubernetes/cloud-provider-openstack/pull/1704/commits/06acf15e30d6bd467d32ff13c4fbbd30ecc9b439.